### PR TITLE
Feature/updates

### PR DESCRIPTION
**Pull request by Bachibouzouk**

### CODE REVIEW

File: soap_api/mastr_power_unit_download.py [hunk 1]
```diff
@@ -73,8 +73,7 @@ def get_power_unit(start_from, limit=API_MAX_DEMANDS):
 
 def download_power_unit(
         power_unit_list_len=2363200,
-        limit=API_MAX_DEMANDS,
-        ofname=None
```
Comment (Bachibouzouk) [inline soap_api/mastr_power_unit_download.py:96]: Please keep the option to give the `ofname` directly as an argument, the tests need it and the user is always free **not** to use it

File: soap_api/mastr_power_unit_download.py [hunk 2]
```diff
@@ -102,10 +99,7 @@ def download_power_unit(
     log.info('Download MaStR Power Unit')
     log.info(f'Number of expected power units: {power_unit_list_len}')
 
-    if ofname is None:
-        ofname = csv_see
```
Comment (Bachibouzouk) [inline soap_api/mastr_power_unit_download.py:125]: Here instead of `csv_see` you can have `fname_all_units`

File: soap_api/utils.py [hunk 3]
```diff
@@ -14,8 +14,31 @@
 log = logging.getLogger(__name__)
 import csv
 
-csv_see_dummy = f'data/bnetza_mastr_{DATA_VERSION}'
-csv_see = f'data/bnetza_mastr_{DATA_VERSION}_power-unit.csv'
+""" dummy stump for other file names """
+fname_dummy = f'data/bnetza_mastr_{DATA_VERSION}'
```
Comment (Bachibouzouk) [inline soap_api/utils.py]: I would prefer `fname_template` for variable name here

Comment (ghost) [reply soap_api/utils.py]: ok! will change

**Pull request merged**